packages/kubernetes-1.16/kubelet-exec-start-conf

```diff
@@ -2,14 +2,12 @@
 ExecStart=
 ExecStart=/usr/bin/kubelet \
 {{~#unless settings.kubernetes.standalone-mode}}
-    --cloud-provider aws \
     --kubeconfig /etc/kubernetes/kubelet/kubeconfig \
 {{~#if (eq settings.kubernetes.authentication-mode "tls")}}
     --bootstrap-kubeconfig /etc/kubernetes/kubelet/bootstrap-kubeconfig \
 {{~/if}}
-{{~else}}
-    --cloud-provider "" \
 {{~/unless}}
+    --cloud-provider "${CLOUD_PROVIDER}" \
```

Comment:
Is there a reason to use an environment variable instead of templating the setting value itself?

Reply:
I was following some existing code, but since it's not necessary I've removed the environment variable.